plugin.xml

```diff
@@ -95,7 +95,7 @@ AND MAY NOT BE RESOLD OR REDISTRIBUTED. USAGE IS BOUND TO THE PSPDFKIT LICENSE A
         <source url="https://github.com/CocoaPods/Specs.git" />
       </config>
       <pods use-frameworks="true">
-        <pod name="PSPDFKit" options="podspec: 'https://customers.pspdfkit.com/cocoapods/YOUR_COCOAPODS_KEY_GOES_HERE/pspdfkit/latest.podspec'" />
+        <pod name="PSPDFKit" options="podspec: 'https://customers.pspdfkit.com/pspdfkit-ios/latest.podspec'" />
```

Comment:
There is a line almost at the bottom of this file, where we show an installation notice and also mention iOS 11. I think updating this would also be nice.

https://github.com/PSPDFKit/PSPDFKit-Cordova/blob/rad/update-for-v9.4/plugin.xml#L109-L110